db/deploy/types/keycloak_jwt.sql

```diff
@@ -25,4 +25,122 @@ create type connectivity_intake_public.keycloak_jwt as (
   broker_session_id text
 );
 
+comment on column connectivity_intake_public.keycloak_jwt.jti is
+  'OPTIONAL - The "jti" (JWT ID) claim provides a unique identifier for the JWT.
```

Comment:
sounds like we might have a bit of tech debt here: required fields should presumably be `not null` on the type itself? Let's resolve this one separately from this PR.

Reply:
Definitely a separate PR since you can't have a constraint on a type:

> The syntax is comparable to CREATE TABLE, except that only field names and types can be specified; no constraints (such as NOT NULL) can presently be included.

If you wanted that constraint in the graphql schema, you could specify it using [a notNull smart tag](https://www.graphile.org/postgraphile/smart-tags/#notnull)

Reply:
I created a new tech debt for this issue.

ref #137